floodwatch/src/js/components/Compare.js

```diff
@@ -64,6 +66,7 @@ export class Compare extends Component {
 type StateType = {
   leftOptions: Array<Filter>,
   rightOptions: Array<Filter>,
+  visibilityMap: VisibilityMap,
   leftData: UnstackedData,
   rightData: UnstackedData,
   currentTopic: string,
```

Comment:
This should be `currentTopic: ?string` if it can be null, which it looks like it can be...